### ISSUE TITLE
system_install.py no longer works with latest ansible

### ISSUE THREAD

**Issue by TomaszKorwel** [cw-ansible/system-install#1]
import part is unable to find libraries:

ansible.utils.template is now ansible.template
ansible.runner.return_data is removed (I do not know at this point how to fix it)

**Comment by renard** [comment]
Which version of ansible do you use?
I haven't play yet with Ansible2.

It works fine with ansible 1.9.4.

Hope that help

**Comment by TomaszKorwel** [comment]
When you grab ansible per your instruction from github you always get the latest one :). For now I took 1.7.1 from archives, works fine. 

Tomasz

**Comment by renard** [comment]
Ok.
Can you just try with latest ansible and commenting out the python imports in line 17 and 18:

``` python
import ansible.utils.template as template
from ansible.runner.return_data import ReturnData
```

I am not sure these are mandatory.

**Comment by TomaszKorwel** [comment]
it fails, throws exception down the line

Tomasz

**Comment by renard** [comment]
Can you send me a full debug please?

**Comment by ssbarnea** [comment]
I am wondering if you managed to find the cause if this error. I got someone else reporting the same kind of error with ansible (on Ubuntu 15.10) and this sounds like a messed ansible installation but I was not able to identify in which context it does happen so I could prevent it from happening.

**Comment by TomaszKorwel** [comment]
I was able to fix the actual error listed above, but I was never able to perform fully automated system provisioning. And since we are relatively small shop with only a handful of new installs a year we've decided to not proceed this route. Instead we simply install base Ubuntu on new machines and run ansible to tailor everything else afterwards.

**Comment by renard** [comment]
Guys please send full debug report as requested 6 months ago.

I use that role on daily tasks without error with ansible 1.9 (I don't have switched to v2 yet). This does not mean the roles is except of bug. But if you found some please send a few details.

Thanks

**Comment by renard** [comment]
closed by 1beef1b0e39144268fbd8f64ad406b10156ec369
